scripts/src/main/resources/scripts/command/ide

```diff
@@ -448,6 +450,33 @@ function doUninstall() {
   doSuccess "Goodbye and have a nice day!"
 }
 
+function doCreate() {
+    local create_path="${1}"
+    if [ -z "${create_path}" ]
+    then
+      create_path="$(pwd)"
+    fi
+
+    if [ ! -d "${create_path}" ]
+    then
+      mkdir -p ${create_path}
+    fi
+
+    count_lines="$(ls -alF ${create_path} | wc -l)"
+    if [ ${count_lines} -gt 3 ]
```

Comment:
The command [ "$(ls -A "${create_path}")" ] && echo "Not Empty" || echo "Empty" runs into an error because of "${create_path}". I not able to use variables in this command.
And I understand if a directory contains no files, it means there are also no .* files.
I tried these (-A option) and find -empty, but nothing works for me in the right way. That is why I preferred my way.